groovy/postInit/chemistry/organic_chemistry/CoalPyrolysisChain.groovy

```diff
@@ -178,4 +178,59 @@ DISTILLATION_TOWER.recipeBuilder()
         .fluidOutputs(fluid('naphthalene_oil') * 200)
         .duration(200)
         .EUt(48)
+        .buildAndRegister()
+
```

Comment:
we should add the dust pyrolysis for other carbon sources like anthracite and coal

Reply:
done